config.md

```diff
@@ -118,6 +118,10 @@ For more information about SELinux, see  [Selinux documentation](http://selinuxp
 * **`noNewPrivileges`** (bool, optional) setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
 [The kernel doc](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) has more information on how this is achieved using a prctl system call.
 
+For Windows-based systems, the process structure supports the following process specific fields:
+
+* **`initialConsoleSize`** (array of uints, optional) specifies the initial console size (height, width) of the terminal if attached.
```

Comment:
Just realized this is `(height, width)` -- isn't it more common for these particular types of geographic values to be ordered `(width, height)`?  Is this ordered this way arbitrarily, or is this how the underlying platform support orders them?

For example:

![image](https://cloud.githubusercontent.com/assets/161631/18562380/4da6f322-7b38-11e6-8e0d-f2767e35ad13.png)